.github/workflows/release-promotion-tests.yml

```diff
@@ -79,8 +79,8 @@ jobs:
           echo ::set-output name=vault-token::$VAULT_TOKEN
           echo ::add-mask::$VAULT_TOKEN
 
-      - name: Render config
-        uses: ./.github/actions/render-config
+      - name: Write configuration
+        uses: ./.github/actions/write-config
         with:
           test-env: ${{ steps.set-env-step.outputs.test-env }}
```

Comment:
Also need to update this step to supply `target` as input, instead of `test-env`